sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -499,7 +499,15 @@ public static UpsertComplexMarketDataRequest ConstantVolSurfaceRequest(DateTimeO
         /// </summary>
         public static UpsertRecipeRequest BuildRecipeRequest(string recipeCode, string scope, ModelSelection.ModelEnum model)
         {
-            var pricingOptions = new PricingOptions(new ModelSelection(ModelSelection.LibraryEnum.Lusid, model));
+            // Note that inside CreatePortfolioAndInstrument, the method TestDataUtilities.BuildInstrumentUpsertRequest books the instrument using "ClientInternal".
+            // As such the quote upserted using "ClientInternal". The market rule key needs to be "ClientInternal" also to find the quote.  
+            var simpleStaticLuidRule = new MarketDataKeyRule(
```

Comment:
Not sure I understand this variable name

Reply:
Ah yes, was when I trying to make it work. Have dropped "luid" in name, should be better. thanks